.github/workflows/package_building.yml

```diff
@@ -16,43 +16,52 @@ on:
   workflow_call:
 
 concurrency:
-  group: pkgdown-${{ github.event.pull_request.number || github.ref }}
+  group: check-test-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
 jobs:
-  pkgdown:
+  check-test:
     name: ${{ vars.CI_IMAGE }}
     runs-on: ubuntu-latest
     container:
       image: ${{ vars.CI_IMAGE }}
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
       - name: Checkout project ⬇️
         uses: actions/checkout@v4
 
-      - name: Install package dependencies 📄
-        uses: boehringer-ingelheim/dv.templates/.github/actions/dependencies@main           
```

Comment:
why is this removed?

Reply:
They are now installed inside single build, but if we prefer a separate step for installing dependencies, no opposition.